src/LogViewer/chunk-worker.js

```diff
@@ -10,6 +10,22 @@ const LINE_CHUNK = 1000;
 const DECODER = new TextDecoder('utf-8');
 // Setting a hard limit on lines since browser have trouble with heights starting at around 16.7 million pixels and up
 const CHUNK_LIMIT = 883000 / LINE_CHUNK;
+// HTML Escape helper utility
+const util = (() => {
+  const reEscape = /[&<>'"]/g;
+  const oEscape = {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    "'": '&#39;',
+    '"': '&quot;'
+  };
+  const fnEscape = (m) => oEscape[m];
+
+  return (Object.freeze || Object)({
```

Comment:
With this now simplified into a few constants, we need a function for doing the raw escape:

```js
const escape = string => string.replace(ESCAPE_REGEX, ESCAPE_FUNCTION);
```